### PR TITLE
 Hybrid Key trust Prerequisites: add prerequisites checklist - PR for Issue #3248

### PR DESCRIPTION
**Pull request by ojrb**
I add the certificate requirements for the Domain Controller, in the Public Key Infrastructure section. @mypil  Please review and Merge

### CODE REVIEW

**Comment by illfated** [comment]
The content text looks OK to me, I have no issues with the grammar or style. Then again, I don't know the topic well enough to confirm the validity of the suggested content to be added.

The only cosmetic change I would like to see, is a more proper PR title. My suggestion is something like

**WHfB Hybrid Key trust Prerequisites: add prerequisites checklist**

(Please also note that issue ticket numbers like #3248 in the PR title or issue ticket title does not link anything on Github and will be a purely visual or cosmetic detail.)

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md [hunk 1]
```diff
@@ -58,7 +58,20 @@ The Windows Hello for Business deployment depends on an enterprise public key in
 
 Key trust deployments do not need client issued certificates for on-premises authentication.  Active Directory user accounts are automatically configured for public key mapping by Azure AD Connect synchronizing the public key of the registered Windows Hello for Business credential to an attribute on the user's Active Directory object.
 
-The minimum required enterprise certificate authority that can be used with Windows Hello for Business is Windows Server 2012.
+The minimum required enterprise certificate authority that can be used with Windows Hello for Business is Windows Server 2012, but you can use a 3rd Party enterprise certification authority too. The detailed requieriments for the Domain Controller certificate are shown below.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md]:
```suggestion
The minimum required enterprise certification authority that can be used with Windows Hello for Business is Windows Server 2012, but you can also use a third party enterprise certification authority. Detailed requirements for Domain Controller certification are as follows:
```

Comment (nenonix) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md]:
```suggestion
The minimum required enterprise certificate authority that can be used with Windows Hello for Business is Windows Server 2012, but you can also use a third-party enterprise certification authority.. The detailed requirements for the Domain Controller certificate are shown below.
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md [hunk 2]
```diff
@@ -58,7 +58,20 @@ The Windows Hello for Business deployment depends on an enterprise public key in
...
+
+* The certificate must have a CRL distribution-point extension that points to a valid certificate revocation list (CRL).
```
Comment (nenonix) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md]:
```suggestion
* The certificate must have a Certificate Revocation List (CRL) distribution-point extension that points to a valid CRL.
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md (hunk 1 above)
Comment (JohanFreelancer9) [reply windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md]:
```suggestion
The minimum required enterprise certificate authority that can be used with Windows Hello for Business is Windows Server 2012, but you can also use a third-party enterprise certification authority. The detailed requirements for the Domain Controller certificate are shown below.

```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md [hunk 3]
```diff
@@ -58,7 +58,20 @@ The Windows Hello for Business deployment depends on an enterprise public key in
...
+* Optionally, the certificate Subject section should contain the directory path of the server object (the distinguished name)
```
Comment (nenonix) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md]:
```suggestion
* Optionally, the certificate Subject section should contain the directory path of the server object (the distinguished name).
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md [hunk 4]
```diff
@@ -58,7 +58,20 @@ The Windows Hello for Business deployment depends on an enterprise public key in
 
 Key trust deployments do not need client issued certificates for on-premises authentication.  Active Directory user accounts are automatically configured for public key mapping by Azure AD Connect synchronizing the public key of the registered Windows Hello for Business credential to an attribute on the user's Active Directory object.
 
-The minimum required enterprise certificate authority that can be used with Windows Hello for Business is Windows Server 2012.
+The minimum required enterprise certificate authority that can be used with Windows Hello for Business is Windows Server 2012, but you can use a 3rd Party enterprise certification authority too. The detailed requieriments for the Domain Controller certificate are shown below.
+
+* The certificate must have a CRL distribution-point extension that points to a valid certificate revocation list (CRL).
+* Optionally, the certificate Subject section should contain the directory path of the server object (the distinguished name) 
+* The certificate Key Usage section must contain:
```
Comment (nenonix) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md]:
```suggestion
* The certificate Key Usage section must contain Digital Signature and Key Encipherment.
```.

Comment (nenonix) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md]:
```suggestion
* The certificate Key Usage section must contain Digital Signature and Key Encipherment.
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md [hunk 5]
```diff
@@ -58,7 +58,20 @@ The Windows Hello for Business deployment depends on an enterprise public key in
...
+Digital Signature, Key Encipherment
+* Optionally, the certificate Basic Constraints section should contain:
```
Comment (nenonix) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md]:
```suggestion
* Optionally, the certificate Basic Constraints section should contain [Subject Type=End Entity, Path Length Constraint=None].
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md [hunk 6]
```diff
@@ -58,7 +58,20 @@ The Windows Hello for Business deployment depends on an enterprise public key in
...
+[Subject Type=End Entity, Path Length Constraint=None]
```
Comment (nenonix) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md]:
```suggestion
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md [hunk 7]
```diff
@@ -58,7 +58,20 @@ The Windows Hello for Business deployment depends on an enterprise public key in
...
+* The certificate Enhanced Key Usage section must contain: Client Authentication (1.3.6.1.5.5.7.3.2) and Server Authentication (1.3.6.1.5.5.7.3.1)
```
Comment (nenonix) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md]:
```suggestion
* The certificate Enhanced Key Usage section must contain Client Authentication (1.3.6.1.5.5.7.3.2) and Server Authentication (1.3.6.1.5.5.7.3.1).
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md [hunk 8]
```diff
@@ -58,7 +58,20 @@ The Windows Hello for Business deployment depends on an enterprise public key in
...
+* The certificate Subject Alternative Name section must contain the Domain Name System (DNS) name.  
+* The certificate template must have an extension that has the BMP data value "DomainController."
```
Comment (nenonix) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md]:
```suggestion
* The certificate template must have an extension that has the BMP data value "DomainController".
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md [hunk 9]
```diff
@@ -58,7 +58,20 @@ The Windows Hello for Business deployment depends on an enterprise public key in
...
+* The domain controller certificate must be installed in the local computer's certificate store
```
Comment (nenonix) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md]:
```suggestion
* The domain controller certificate must be installed in the local computer's certificate store.
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md (hunk 1 above)
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md]:
```suggestion
The minimum required enterprise certification authority that can be used with Windows Hello for Business is Windows Server 2012, but you can also use a third-party enterprise certification authority. The detailed requirements for the Domain Controller certificate are shown below.
```

**Comment by ojrb** [comment]
@mypil I accept all the suggestions ! please review and merge !

**Comment by mypil** [comment]
@mapalko - Can you please assist to check, approve and merge the content for this PR based on issue #3248?

Thank you.

**Comment by mypil** [comment]
@mapalko - I would like to follow up on this. Can you please assist to check, approve and merge the content for this PR based on issue #3248?

Thank you.

**Comment by illfated** [comment]
Looking back at the Github source page ([hello-hybrid-key-trust-prereqs.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md)), I wonder if maybe @Justinha , @DaniHalfin or even @lizap could help out merging this PR, just in case @mapalko is unavailable for the time being.

**Comment by mypil** [comment]
@mapalko @DaniHalfin @lizap @Justinha @greg-lindsay @mjcaparas @AndreaBarr -  would like to follow up on this. Can anyone of you please assist to check, approve and merge the content for this PR based on issue #3248?

Thank you.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md [hunk 10]
```diff
@@ -58,7 +58,18 @@ The Windows Hello for Business deployment depends on an enterprise public key in
 
 Key trust deployments do not need client issued certificates for on-premises authentication.  Active Directory user accounts are automatically configured for public key mapping by Azure AD Connect synchronizing the public key of the registered Windows Hello for Business credential to an attribute on the user's Active Directory object.
 
-The minimum required enterprise certificate authority that can be used with Windows Hello for Business is Windows Server 2012.
+The minimum required enterprise certificate authority that can be used with Windows Hello for Business is Windows Server 2012, but you can also use a third-party enterprise certification authority. The detailed requirements for the Domain Controller certificate are shown below.
+
+* The certificate must have a Certificate Revocation List (CRL) distribution point extension that points to a valid CRL.
+* Optionally, the certificate Subject section should contain the directory path of the server object (the distinguished name).
+* The certificate Key Usage section must contain Digital Signature and Key Encipherment.
+* Optionally, the certificate Basic Constraints section should contain:
```
Comment (nenonix) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md]:
```suggestion
* Optionally, the certificate Basic Constraints section should contain [Subject Type=End Entity, Path Length Constraint=None].
```

**Comment by mypil** [comment]
@ojrb - please check on the additional feedback from @nenonix  and make the necessary updates. Thanks.

**Comment by ojrb** [comment]
@mypil based on @nenonix recommendations I created a new PR #3827 with the missing change, after the merge !

**Pull request merged**